blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -3,29 +3,28 @@ import ReactDOM from 'react-dom';
 import Extensions from '@jenkins-cd/js-extensions';
 
 import {
+    buildClassicInputUrl,
```

Comment:
Feel free to stomp this file - Ill come back and update the changes to `QueuedState` when your branch is merged